File: tests/fixtures/ruff-grammar/grammar.py
import os  # NOQA: F401
import sys  # noqa:F401,E402
import json  # noqa: F401 E402
import re  # type: ignore  # noqa: F401  # embedded after another directive
import io  # noqa: F401, oops
import abc  # noqa
NAMES = (os, sys, json, re, io, abc, "# noqa: F811")

# llmlint: ignore-file[suppressions_justified] fixture input, not production code: these lines exercise every directive form the parser claims, reason-less ones included, and the e2e asserts each is reported exactly as written.

File: tests/fixtures/eslint-parity/grammar.js
/* eslint-disable no-alert -- this prelude is interactive on purpose */
alert("inside the block");
/* eslint-enable no-alert */
console.log("trailing"); // eslint-disable-line no-console -- the trailing form
// eslint-disable-next-line no-console -- the next-line form
console.log("next line");
/* eslint-disable-next-line no-console
   -- a reason that spans
      several lines */
console.log("multi line");
/* eslint-disable */
console.log("everything from here on");
alert("including this");

// llmlint: ignore-file[suppressions_justified] fixture input, not production code:
// these lines exist to exercise every directive form the parser claims, including
// the blanket `eslint-disable` whose reason-less shape the e2e asserts is reported.
